ls1intum/Artemis
# Make summary available for instructors

### Checklist

### Motivation and Context


The instructor could not view the whole summary of a student exam

### Description

Added a new page where the instructor can view a student exam

### Steps for Testing


1. Log in to Artemis
2. Navigate to the exam view of a instructor
3. Open a student exam
4. Click on **summary**
5. Check that the summary is displayed correctly

### Screenshots


<img width="916" alt="Bildschirmfoto 2020-07-03 um 14 16 50" src="https://user-images.githubusercontent.com/51987021/86468633-1bc5fd80-bd38-11ea-8bb5-d13a8e154f1b.png">
